templates/service.cmd.j2

```diff
@@ -0,0 +1 @@
+java.exe -Xrs {{ slave_windows_java_opts }} -jar "{{ slave_windows_workdir }}/slave.jar" -jnlpUrl {{ master_url }}/computer/{{ slave_agent_name }}/slave-agent.jnlp -secret {{ jenkins_slave_secret }}
```

Comment:
Double quotes are dangerous for using since any backslash in a path is treated like escape symbol. Please, use single quotes.

Reply:
fixed